llama_stack/apis/tools/rag_tool.py

```diff
@@ -72,6 +72,7 @@ class RAGQueryConfig(BaseModel):
     query_generator_config: RAGQueryGeneratorConfig = Field(default=DefaultRAGQueryGeneratorConfig())
     max_tokens_in_context: int = 4096
     max_chunks: int = 5
+    mode: str | None = None
```

Comment:
you'll need to rebase my latest changes here and can you add this to the docstring? also you'll need to regenerate the API documentation using `uv run --with ".[dev]" ./docs/openapi_generator/run_openapi_generator.sh`